website/docs/reference/resource-properties/argument-type.md

```diff
@@ -23,6 +23,14 @@ macros:
 ## Definition
 The data type of your argument. Note that this is only used for documentation purposes — there are no restrictions on the values you can use here.
 
+:::tip
+From dbt Core v1.10, you can opt into validating the arguments you define in macro documentation using the `validate_macro_args` behavior change flag. When enabled, dbt will:
+- Warn if documented argument names don’t match the macro definition.
+- Warn if `type` fields don’t follow supported formats (like `string`, `integer`, `optional[list[str]]`, and so on).
```

Comment:
I don't think we can just say "and so on" here - I think this is the place to list out the accepted values for `type`

Reply:
OK request to add accepted values here: https://github.com/dbt-labs/docs.getdbt.com/pull/7128#discussion_r2025509505